examples/apache-directory-list-local/webdav.conf

```diff
@@ -13,4 +13,15 @@
         Options +Indexes
         HeaderName /webdav-js/examples/apache-directory-list-local/header.html
     </Location>
+
+    # Add security
+    <Directory /var/www/webdav-js>
+        Options +Indexes
+        DAV On
```

Comment:
Or this! I removed both in some tests and it appeared to have no impact.

Reply:
that's because of dockers `:ro`(read only) flag this will not be the case on production

Reply:
Oh you mean the `Options +Indexes` `DAV On`

Reply:
Tested works fine with a usual 
```
[Tue Nov 13 15:04:25.048605 2018] [access_compat:error] [pid 152:tid 139632591230720] [client 1
72.17.0.1:44438] AH01797: client denied by server configuration: /var/www/webdav-js
```